lib/active_model/serializable/utils.rb

```diff
@@ -4,13 +4,15 @@ module Utils
       extend self
 
       def _const_get(const)
-        begin
-          method = RUBY_VERSION >= '2.0' ? :const_get : :qualified_const_get
-          Object.send method, const
-        rescue NameError
-          const.safe_constantize
+        Serializer.serializers_cache.fetch_or_store(const) do
```

Comment:
Yeah. I'd duplicate the cache call to avoid coupling between the utils model and the `Serializer` constant.